ideascube/library/tests/test_migrations.py

```diff
@@ -20,6 +20,126 @@
 }
 
 
+@migration_test(
+    migrate_from=[
+        ('library', '0007_auto_20161007_1220'),
+        ('monitoring', '0004_auto_20161007_1240'),
+        ('taggit', '0002_auto_20150616_2121'),
+    ], migrate_to=[('library', '0008_library_stock')])
+def test_library_to_monitoring_with_duplicated_data(migration):
```

Comment:
A global comment for this test.

- Please use more explicit variable name than `book`, `item`, `specimen` when you're testing that migration had run correctly.
- Please test the books in the same order you've create them (this migration is far complex enough without that)